tests/manifold/spherical_manifold_04.output

```diff
@@ -0,0 +1,3 @@
+
+DEAL::Volume:       14109.1
```

Comment:
As you prefer. If you want to see a better convergence, I can put a `4`...

:)